ocaml/xapi/xapi_xenops.ml

```diff
@@ -2413,7 +2413,11 @@ let suspend ~__context ~self =
 			(* XXX: this needs to be at boot time *)
 			let space_needed = Int64.(add (of_float (to_float vm_t.Vm.memory_static_max *. 1.2 *. 1.05)) 104857600L) in
```

Comment:
It's a bit out of scope for this PR, but I'd prefer to see some of the magic numbers in this expression defined somewhere...